packages/client/wallets/aa/src/utils/signer.ts

```diff
@@ -81,16 +55,23 @@ export async function createOwnerSigner({
             idToken: signer.jwt,
         });
 
-        return getRPCProviderOwner(provider);
+        const web3 = new Web3(provider as any);
+        const [address] = await web3.eth.getAccounts();
+
+        return await providerToSmartAccountSigner(provider! as EIP1193Provider, address as Hex);
```

Comment:
we're using `as any` and `!` to circumvent better `null` handling. 
Can we add
```suggestion
        if (provider == null) {
            throw new WalletSdkError("web3auth returned a null signer")
        }
        
        const web3 = new Web3(provider);
        const [address] = await web3.eth.getAccounts();
        return await providerToSmartAccountSigner(provider as EIP1193Provider, address as Hex);
```